app/server/openshift/src/main/java/io/syndesis/server/openshift/OpenShiftConfigurationProperties.java

```diff
@@ -47,6 +47,9 @@
 
     private boolean debug;
 
+    private int maximumRetries = 3;
+    private long pollingInterval = 5000;
```

Comment:
Maybe call it 'retryInterval' as its about retries ? Actual, when I saw polling the first time I thought is about a regular, periodic polling mechanism. But its not, it only kicks in when a call fails.

Reply:
@rhuss: Actually we DO poll the until the build/deployment is complete or until a timeout is reached. For each iteration that fails we increase a counter and if the maxium threshold is reached we just bail. So from my point of view the name has to stay as is.